WallsAndHoles/editor.h

```diff
@@ -5,6 +5,7 @@
 #include "tilemaptoolmanager.h"
 #include "tiletemplateset.h"
 #include "mapview.h"
+#include "xmltool.h"
```

Comment:
Technically, since this header is only used in editor.cpp, it would be better to put it there.